portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,70 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  font: 22px 'Manrope', sans-serif;
+
+  padding: 220px;
+  text-align: center;
+}
+
+#intro {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+#headshot {
+    border-radius: 100%;
+    padding: 30px;
+    width: 200px;
+}
+
+#name {
+    font: bold 80px 'Gochi Hand', cursive;
+}
+
+#about {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    margin: 100px;
+    text-align: left;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#info {
```

Comment:
With the change from id to class that I mentioned in index.html, this would become:
.info {